ui/scripts/instances.js

```diff
@@ -3437,6 +3437,13 @@
 						custom: cloudStack.uiCustom.granularDetails({
                         resourceType: 'UserVm',
 							dataProvider: function(args) {
+                                // no paging for listVirtualMachines details
+                                if (args.page > 1){
+                                    args.response.success({
+                                        data: []
+                                    });
+                                    return;
+                                }
```

Comment:
Can you fix indentation @Spaceman1984 ?

Reply:
What do you mean @anuragaw ? I see no problem with the indentation here.

Reply:
Isn't this supposed to be inside dataProvider @DaanHoogland ?
![image](https://user-images.githubusercontent.com/43956255/71301851-6a2e3300-23ca-11ea-9eaa-1f435ec3b363.png)

Reply:
The indentation is fine when viewing with vscode and intellij, I think this is a github display issue.

Reply:
What you see in IDEs is actually rendered as per configuration of whitespace. However, this is old UI and I've seen this broken at many files.